service/header_service.go

```diff
@@ -66,12 +66,12 @@ func (hs *HeaderService) IsCurrent() bool {
 	// Not current if the latest main (best) chain height is before the
 	// latest known good checkpoint (when checkpoints are enabled).
 	checkpoints := hs.checkpoints
-	checkpoint := &checkpoints[len(checkpoints)-1]
+	checkpoint := checkpoints[len(checkpoints)-1]
```

Comment:
What if the `checkpoints` is empty?

Reply:
I have no idea what are you talking about :P Change reverted - out of the scope